alice/chain/measure.go

```diff
@@ -27,13 +28,13 @@ type base struct {
 // router.Get("/user", b.Measure("get users", user.Get()))
 //
 func NewBase(b alice.Chain, timer middleware.Timer, logger middleware.Logger) Measurer {
-	c := b.Append(middleware.Recovery, middleware.UserIDInjector, middleware.RequestID, logger.Log)
+	c := b.Append(middleware.Recovery, middleware.NewUserIDInjector(jwtdecode.NewJWTDecoder()).Inject, middleware.RequestID, logger.Log)
```

Comment:
thoughts on keeping backwards compatibility here or pushing this out to the routes.go files so they have to pass in a JWTDecoder ?